packages/block-library/src/accordion-header/save.js

```diff
@@ -12,20 +8,18 @@ import {
 } from '@wordpress/block-editor';
 
 export default function save( { attributes } ) {
-	const { level, title, iconPosition, textAlign, showIcon } = attributes;
+	const { level, title, iconPosition, showIcon } = attributes;
 	const TagName = 'h' + level;
 
 	const blockProps = useBlockProps.save( {
-		className: clsx( 'accordion-content__heading', {
-			[ `has-text-align-${ textAlign }` ]: textAlign,
-		} ),
+		className: 'accordion-content__heading',
```

Comment:
Nit: The `accordion-content__heading` class is not used anywhere, so you can probably delete it. This CSS class exists in both the Edit component and the save function.